etl/run_sql/run_sql.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+import json
+import logging
+import os
+import sys
+
+import requests
+
+from utils.cli import get_cli_args
+from utils.commands import COMMANDS
+
+ENDPOINT = os.getenv("HASURA_GRAPHQL_ENDPOINT")
+ADMIN_SECRET = os.getenv("HASURA_GRAPHQL_ADMIN_SECRET")
+
+ENDPOINT="http://localhost:8084/v2/query"
+
+logging.basicConfig(stream=sys.stdout, level=logging.INFO)
+
+
+def run_sql(sql):
+    """Run arbitrary SQL through the the Hasura schema API.
+
+    The ENDPOINT must provide the full path to the schema API, which
+    is currently `/v2/query`.
+
+    Args:
+        sql (Str): The sql to be executed, including a final semicolon.
+    """
+    if not ENDPOINT:
+        raise OSError("HASURA_GRAPHQL_ENDPOINT env var is required")
```

Comment:
I think you may have two conflicting variables here.

Reply:
yep, that's right. i've just run into this situation before and found it a tad confounding to debug 😅